resources/views/events/admin/create.blade.php

```diff
@@ -111,7 +111,8 @@
             <div class="md:w-2/3">
                 <div>
                     <input type="datetime-local" name="last_cancel_date" class="form-input w-full"
-                           required value="{{ old('last_cancel_date') }}">
+                           value="{{ old('last_cancel_date',
+                                $event->last_cancel_date ? \App\Utils\Date::fromString($event->last_cancel_date)->tz(config('app.timezone'))->format('Y-m-d\\TH:i') : null) }}">
```

Comment:
Event is null as this is the `create` view

Reply:
My bad as I just copied code without full manual testing, committed ae81471